client-course-schedulizer/src/components/Tabs/Harmony/Harmony.tsx

```diff
@@ -0,0 +1,45 @@
+import { Harmony as H } from "@harmoniously/react";
+import { Assignments } from "harmoniously";
+import React, { useContext } from "react";
+import { AppContext } from "utilities/contexts";
+
+// TODO: allow this to be edited by users.
+const times = ["mwf800", "mwf900", "mwf1030", "mwf1130", "mwf1230", "mwf130", "mwf230", "mwf330"];
+
+// temporary hook used to get data from the imported schedule and generate fake assignments.
+const useInferredAssignments = () => {
+  const {
+    appState: { rooms, professors, classes },
+  } = useContext(AppContext);
+
+  const inferredAssignments: Assignments = {};
+  classes.forEach((cls) => {
+    inferredAssignments[cls] = {
+      // TODO: update this. https://stackoverflow.com/a/5915122/9931154
+      professor: professors[Math.floor(Math.random() * professors.length)],
+      rooms: [...rooms, "a", "b"],
```

Comment:
What is the reason for adding "a" and "b" to the list of possible rooms?

Reply:
This is just to make there are enough rooms to find a schedule, I'll remove it in the future. This is just for testing.